src/ai_config/groqConfig.js

```diff
@@ -10,12 +10,16 @@ import path from 'path';
 const configFilePath = path.join(os.homedir(), '.gimme_readme_config');
 dotenv.config({ path: configFilePath });
 
-// Initialize groq AI client
-const groq = new Groq({ apiKey: process.env.GROQ_KEY });
+
 
 // Export function to handle Groq-specific prompting
-export async function promptGroq(prompt, model, temperature = 0.5) {
+export async function promptGroq(prompt, model, temperature = 0.5, apiKey = "") {
```

Comment:
Similar to above, passing the apiKey as an argument shouldn't be necessary as I don't want to allow the user to pass their api keys in the command-line. Each of the "prompt" functions can look in the `.gimme_readme_config` file directly for the correct API keys to use